carparkmanager-persistence/src/main/java/cz/muni/fi/pa165/carparkmanager/persistence/dao/EmployeeDao.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cz.muni.fi.pa165.carparkmanager.persistence.dao;
+
+import cz.muni.fi.pa165.carparkmanager.persistence.entity.Employee;
+import enums.ClassificationOfEmployee;
+import java.util.List;
+
+/**
+ *
+ * @author xbonco1
+ */
+public interface EmployeeDao {
+    /**
+     * Method to create an employee
+     * @param employee
+     * @return a new Employee
+     * @throws IllegalArgumentException 
+     */
+    
+    public Employee create (Employee employee) throws IllegalArgumentException;
```

Comment:
podla mna by to mohlo byt skor void

Reply:
som sa nevedel uplne rozhodnut medzi voidom, longom co by ti vratilo Id vytvoreneho a konkretnym objektom - dal som objekt, ale nieje problem zmenit

Reply:
z miesta, kde budes volat ten create(), musis mat vytvoreny objekt Employee a v tom objekte sa po me.persist() automaticku zmeni ID, cize ho nepotrebujes vracat